migration/iD10/oracle/202207081747_IDEMPIERE-5302.sql

```diff
@@ -0,0 +1,911 @@
+-- IDEMPIERE-5302
+SELECT register_migration_script('202207081747_IDEMPIERE-5302.sql') FROM dual;
+
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- Jul 12, 2022, 9:14:27 AM CEST
+INSERT INTO AD_Element (AD_Element_ID,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,ColumnName,Name,Description,PrintName,EntityType,AD_Element_UU) VALUES (203568,0,0,'Y',TO_TIMESTAMP('2022-07-12 09:14:27','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2022-07-12 09:14:27','YYYY-MM-DD HH24:MI:SS'),100,'IsShowInDrillOptions','Show In Drill Options','This parameter enables the table to be displayed in Drill Assistant - Table tab','Show In Drill Options','D','6a1f3c17-a045-4c39-9cb9-ecebb729fc85')
+;
+
+-- Jul 12, 2022, 9:32:52 AM CEST
+INSERT INTO AD_Column (AD_Column_ID,Version,Name,Description,AD_Table_ID,ColumnName,DefaultValue,FieldLength,IsKey,IsParent,IsMandatory,IsTranslated,IsIdentifier,SeqNo,IsEncrypted,AD_Reference_ID,AD_Client_ID,AD_Org_ID,IsActive,Created,CreatedBy,Updated,UpdatedBy,AD_Element_ID,IsUpdateable,IsSelectionColumn,EntityType,IsSyncDatabase,IsAlwaysUpdateable,IsAutocomplete,IsAllowLogging,AD_Column_UU,IsAllowCopy,SeqNoSelection,IsToolbarButton,IsSecure,FKConstraintType,IsHtml) VALUES (214721,0,'Show In Drill Options','This parameter enables the table to be displayed in Drill Assistant - Table tab',100,'IsShowInDrillOptions','N',1,'N','N','N','N','N',0,'N',20,0,0,'Y',TO_TIMESTAMP('2022-07-12 09:32:52','YYYY-MM-DD HH24:MI:SS'),100,TO_TIMESTAMP('2022-07-12 09:32:52','YYYY-MM-DD HH24:MI:SS'),100,203568,'Y','N','D','N','N','N','Y','ab5fc669-6971-4f08-a7f3-160796cc2a36','Y',0,'N','N','N','N')
+;
+
+-- Jul 12, 2022, 9:32:57 AM CEST
+ALTER TABLE AD_Table ADD IsShowInDrillOptions CHAR(1) DEFAULT 'N' CHECK (IsShowInDrillOptions IN ('Y','N'))
```

Comment:
The default of the new column must be Y ???   For backward compatibility?

If yes, please apply the following patch:
[5302_fixdefault.txt](https://github.com/idempiere/idempiere/files/9093388/5302_fixdefault.txt)

However, I don't see a difference on this.

In my tests I generate a report from Business Partner and the Drill list is disappeared, before this patch the Drill list is pre-filled with the defaults discovered by the system.
Is this feature supposed to disappear?

Reply:
Yes, we removed the Drill list, it was requested in the last pull request. The IsShowInDrillOptions column now only defines, which tables will appear in the Table tab of Drill Assistant. We also agreed in the previous PR, that it should be default 'N', and set 'Y' only on specific tables. There is an SQL UPDATE related to this in the migration scripts (line 28 in oracle; 25 in postgres)

Reply:
> Yes, we removed the Drill list, it was requested in the last pull request. The IsShowInDrillOptions column now only defines, which tables will appear in the Table tab of Drill Assistant. We also agreed in the previous PR, that it should be default 'N', and set 'Y' only on specific tables. There is an SQL UPDATE related to this in the migration scripts (line 28 in oracle; 25 in postgres)

No, I just checked the original pull request 1345 and the default of IsShowInDrillOptions is set to N (line 19 oracle) and some lines later is set to Y (line 26 oracle).

For the deprecation of the old drill functionality I'll move the discussion to the ticket.

Reply:
I see, OK util then I'm fixing IsShowInDrillOptions, default to 'Y'.